content/en/hosting/analytics/setup-docker-compose.md

```diff
@@ -1,29 +1,34 @@
 ---
-title: "Local CHT Sync Setup"
-weight: 2
-linkTitle: "Local CHT Sync Setup"
+title: "CHT Sync Setup with Docker Compose"
+weight: 5
+linkTitle: "Setup with Docker Compose"
 description: >
-  Setting up a local deployment of CHT Sync with the CHT
+  Setting up CHT Sync with Docker Compose and the CHT
 relatedContent: >
   core/overview/architecture
   core/overview/cht-sync
 aliases:
    - /apps/guides/data/analytics/setup
 ---
 
-Before setting up CHT Sync in production, it's very handy to be able to run it locally. This will allow you to experiment with the data flow and easily query development data quickly and locally. 
+This guide will walk you through setting up a deployment of CHT Sync with the CHT using Docker Compose.
 
-These instructions assume you're running CHT Sync, CHT Core and PostgreSQL either locally on your workstation or on a local server. They are not meant to be used to deploy a secure, always on production instance.
```

Comment:
hey - this is great!  I think with CHT Core's move to support docker in production, it makes sense to have parallel support over here in data sync and analytics.  user's can decide if this is for prod or a quickie dev setup!